src/map/clif.c

```diff
@@ -11065,7 +11065,8 @@ static void clif_parse_LoadEndAck(int fd, struct map_session_data *sd)
 		// Notify everyone that this character logged in. [Skotlex]
 		map->foreachpc(clif->friendslist_toggle_sub, sd->status.account_id, sd->status.char_id, 1);
 
-		clif->open_ui_send(sd, ZC_TIPBOX_UI);
+		if (battle_config.show_tip_window != 0)
+			clif->open_ui_send(sd, ZC_TIPBOX_UI);
```

Comment:
error from ci ``clif.c:11069:27: error: use of undeclared identifier 'ZC_TIPBOX_UI'``
this is because old packet version too

Reply:
Thanks. Fixed, too.